[DE] Add `osu!stream` (#6348)
ppy/osu-wiki: wiki/osu!stream/de.md

# osu!stream

![](img/Os-Logo.jpg "Logo von osu!stream")

> **osu!stream** ist eine Version von osu!, die von Grund auf neu entwickelt wurde und extra für berührungssensitive Geräte ausgelegt ist. Aktuell ist sie nur für Apples [iOS](http://en.wikipedia.org/wiki/IOS)-Plattform verfügbar. Ein [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29)-Port war geplant und beinahe fertig, wurde aber nicht veröffentlicht, da es Probleme bei der Audiolatenz gab. Dadurch wurde der Spielspaß massiv beeinträchtigt.

Reviewer: ```suggestion
**osu!stream** ist eine Version von osu!, die von Grund auf neu entwickelt wurde und extra für berührungssensitive Geräte ausgelegt ist. Aktuell ist sie nur für Apples [iOS](http://en.wikipedia.org/wiki/IOS)-Plattform verfügbar. Ein [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29)-Port war geplant und beinahe fertig, wurde aber nicht veröffentlicht, da es Probleme bei der Audiolatenz gab, die den Spielspaß massiv beeinträchtigen.
```

Your version sounds to me like the fact that there is no Android port released is inhibiting enjoyment.

Author: I agree with you, but I would choose `beeinträchtigten`, because the rest of the sentence is written in the past tense too.

Reviewer: Sure.

> Enthalten ist ein komplett überarbeitetes Spielsystem mit neuen Spielelementen, wie den Hold-Circles und verbundenen Beats. Klassische osu! Beatmaps werden nicht ohne manuelle Umwandlung unterstützt. Stattdessen wurden neue Beatmaps explizit von einem besonderen Team an Mappern erstellt, um die Chance der neuen Spielelemente auszunutzen. Diese neuen Beatmaps haben einen [strikteren Standard](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).

Reviewer 2: I'm not too sure about this one, but it feels kinda wrong. I'd put the comma here instead:
```suggestion
Enthalten ist ein komplett überarbeitetes Spielsystem, mit neuen Spielelementen wie den Hold-Circles und verbundenen Beats. Klassische osu! Beatmaps werden nicht ohne manuelle Umwandlung unterstützt. Stattdessen wurden neue Beatmaps explizit von einem besonderen Team an Mappern erstellt, um die Chance der neuen Spielelemente auszunutzen. Diese neuen Beatmaps haben einen [strikteren Standard](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```
Or simply no comma at all:
```suggestion
Enthalten ist ein komplett überarbeitetes Spielsystem mit neuen Spielelementen wie den Hold-Circles und verbundenen Beats. Klassische osu! Beatmaps werden nicht ohne manuelle Umwandlung unterstützt. Stattdessen wurden neue Beatmaps explizit von einem besonderen Team an Mappern erstellt, um die Chance der neuen Spielelemente auszunutzen. Diese neuen Beatmaps haben einen [strikteren Standard](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```
Feel free to disregard it.

> [osu!stream wurde ausführlich behandelt im osu!monthly \#5 mit einem Interview.](https://osu.ppy.sh/community/forums/topics/59924)

Author: The `\` needs to be there because it also is in the English version. But otherwise, I agree with your suggestion.

> [osu!stream wurde ausführlich behandelt im osu!monthly \#5 mit einem Interview.](https://osu.ppy.sh/community/forums/topics/59924)

Reviewer 3: oh whoops i rewrote the entire sentence didnt see i missed the `\`

> Die Entwicklung für diese Version endete in 2020. Für mehr Informationen, lies den [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) über ihre letzte Veröffentlichung. Der Quellcode zusammen mit den Beatmap-Konvertierungs- und Test-Tools wurde im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kostenpflichtigen Beatmaps sind jetzt kostenlos im in-game "Store" erhältlich.

Reviewer 2: ```suggestion
Die Entwicklung für diese Version endete in 2020. Für mehr Informationen lies den [Blog-Beitrag](https://blog.ppy.sh/osu-stream-2020-release/) über ihre letzte Veröffentlichung. Der Quellcode zusammen mit den Beatmap-Konvertierungs- und Test-Tools wurde im selben Jahr unter einer freien Lizenz auf [GitHub](https://github.com/ppy/osu-stream) zur Verfügung gestellt. Alle zuvor kostenpflichtigen Beatmaps sind jetzt kostenlos im in-game "Store" erhältlich.
```
Not sure about this one either, but I think `Für mehr Informationen` is not a clause, hence it shouldn't be separated by a comma.

  - Wenn nach der Apple ID gefragt wird, gebe die Apple ID und das Passwort ein.
- Stecke dein iDevice in dein Gerät (welches osu!stream hat).
- Transferiere die osu!stream Dateien mit iTunes.
- Entferne das iDevice und osu!stream kann gespielt werden.

## Beatmaps hinzufügen

**Versichere dich, dass das iDevice sich mit dem Internet verbinden kann (entweder über WLAN oder 3G/4G).**

- Öffne osu!stream.
- Drücke auf das osu! Symbol und du solltest direkt in das Hauptmenü weitergeleitet werden (oder das Tutorial beim ersten Mal)
- Drücke die "Store" Leiste
  - Alternativ kannst du auch in der "Play" Sektion auf "Download more songs!" drücken.
- Wähle dein Wunschpaket.

## Was ist der Unterschied?

- **Nur auf iOS-Geräten spielbar.** (aktuell)
- Beatmaps sind von Grund auf neu gemacht.
- Es wird kein Account zum Herunterladen von Songpaketen benötigt.
- osu!stream kann mit einem Twitterkonto genutzt werden.
- Höchstpunktestände dürfen anonym zur Seite als "Guest" gesendet werden (benötigt eine Internetverbindung).
- Der Circle-Timer wird ersetzt und befindet sich nun unten mit einer kleinen Grenzlinie, die abhängig von der Genauigkeit des Spielverlaufs gefärbt ist. \[Bottom(Long)\]

> - Die **Durchschnittliche Zeit** wird am Ergebnisbildschirm angezeigt.

Reviewer: ```suggestion
- Die **Durchschnittliche Abweichung** wird am Ergebnisbildschirm angezeigt.
```

Assuming this is referring to the deviation of hits from the  "perfect" time. Using just `Zeit` would in my mind refer to the runtime of the beatmap.

> - Alte Aufzeichnungen können nicht gespeichert werden.

Reviewer: ```suggestion
- Alte Rekorde und Aufzeichnungen können nicht gespeichert werden.
```

- Der Skin kann nicht gewechselt werden und Mods sind nicht auswählbar (bislang).
- Easter Eggs können gefunden werde, aber es gibt noch keine Seite für Errungenschaften...

### Neues Spielsystem

**Information:** Teste die neuen Spielelemente im "Tutorial" Bildschirm.

#### Fingerführungsanzeige

![](img/Os-FG.jpg "Fingerführungsanzeige")

- Die zwei transparenten Kreise, die im Spiel auftauchen, helfen Anfängern zu wissen, wann sie die Objekte treffen müssen.
  - Es zeigt die empfohlene Methode, um die Map perfekt abzuschließen.

> - TDie grüne Farbe ist für die linke Hand und die rote Farbe ist für die rechte Hand.

Reviewer: ```suggestion
- Die grüne Farbe ist für die linke Hand und die rote Farbe ist für die rechte Hand.
```

Die Hintergrundfrabe ist grün.

#### Streammodus

![](img/Os-ModeStream.jpg "Streammodus")

**Schwierigkeitsgrad: Easy ("Stream Down!" von Normal aus) ~ Normal (Startpunkt) ~ Hard ("Stream Up!" von Normal aus)**

Der Streammodus ist ein exclusiver Modus in osu!stream. In diesem Modus starten Spieler mit einer zur Hälfte gefüllten "zweiten" Lebensleiste und der normale Modus ist eingestellt. Der Spieler muss darauf die Lebensleiste füllen, um "Stream Up!" zu aktivieren, was im Grunde die Schwierigkeit erhöht (durch das Hinzufügen von weiteren Objekten und dem Wechsel zum schweren Schwierigkeitsgrad der Map). Dies ist aber immer noch bedeutend einfacher als der Expertenmodus. Wenn der Spieler in den "Stream Up!" Modus gelangt, dann wird eine Lebensleiste vergeben. Sollte diese Lebensleiste leer werden, wird "Stream Down!" erscheinen. Der Schwierigkeitsgrad geht danach zurück auf Normal oder Leicht, abhängig von der Anzahl der Lebensleisten.

Lange Rede kurzer Sinn, der Spieler startet zwischen Leicht und Schwer.

> Beachte, dass der Spieler **mindestens ein A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Wenn die Genauigkeit kleiner als 80% beträgt, egal ob es eine volle Combo war, wird kein A vergeben (Es gilt meistens als B).

Reviewer 3: ```suggestion
Beachte, dass der Spieler **mindestens ein A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Eine Genauigkeit von weniger als 80%, egal ob es eine volle Combo war, zählt nicht als A (Es gilt dann wie üblich als B).
```

> Beachte, dass der Spieler **mindestens ein A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Wenn die Genauigkeit kleiner als 80% beträgt, egal ob es eine volle Combo war, wird kein A vergeben (Es gilt meistens als B).

Reviewer: ```suggestion
Beachte, dass der Spieler **mindestens einen A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Wenn die Genauigkeit kleiner als 80% beträgt, egal ob es eine volle Combo war, wird kein A vergeben (Es gilt meistens als B).
```

> Beachte, dass der Spieler **mindestens einen A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Eine Genauigkeit von weniger als 80%, egal ob es eine volle Combo war, zählt nicht als A (Es gilt dann wie üblich als B).

Reviewer 2: ```suggestion
Beachte, dass der Spieler **mindestens einen A Rang (>80% Genauigkeit) in diesem Modus haben muss, um den Expertenmodus des Songs freizuschalten**. Eine Genauigkeit von weniger als 80%, auch wenn es eine volle Combo war, zählt nicht als A (Es gilt dann wie üblich als B).
```
Seems better to me

Die Hintergrundfarbe ist grün (Leicht), blau (Normal), und pink (Schwer).

---

![](img/Os-SU.png "Stream Up!")

---

![](img/Os-SD.png "Stream Down!")

---

> ![](img/Os-Zero1.png "Lebenspunkte an einem kritischen Punkt")

Reviewer 3: ```suggestion
![](img/Os-Zero1.png "Kritische Gesundheit")
```
maybe a bit more direct translation, doesnt really matter tho

---

> ![](img/Os-Zero2.png "Spiel vorüber")

Reviewer 3: ```suggestion
![](img/Os-Zero2.png "Spiel vorbei")
```
maybe

---

#### Expertenmodus

![](img/Os-ModeExpert.jpg "Expertenmodus")

**Schwierigkeitsgrad: Experte**

> Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar. Der Expertenmodus ist für Spieler, die eine größere Herausforderung suchen. In diesem Modus erhalten die Spieler einen vollen Lebensbalken und müssen ihn bis zum Ende des Liedes erhalten. Der erfolgreiche Abschluss schaltet nichts frei außer das Recht zum Angeben.

Reviewer 2: ```suggestion
Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar. Der Expertenmodus ist für Spieler, die eine größere Herausforderung suchen. In diesem Modus erhalten die Spieler einen vollen Lebensbalken und müssen ihn bis zum Ende des Liedes erhalten. Der erfolgreiche Abschluss schaltet nichts frei außer dem Recht zum Angeben.
```
I think 🤔

> Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar, der Expertenmodus ist für alle, die eine größere Herausforderung suchen. In diesem Modus bekommen Spieler eineinhalb Lebensleisten und müssen diese bis zum Ende des Songs erhalten. Der erflogreiche Abschluss schaltet nicht frei außer das Recht zum Angeben.

Reviewer 3: ```suggestion
Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar. Der Expertenmodus ist für Spieler, die eine größere Herausforderung suchen. In diesem Modus erhalten die Spieler einen vollen Lebensbalken und müssen ihn bis zum Ende des Liedes erhalten. Der erflogreiche Abschluss schaltet nicht frei außer das Recht zum Angeben.
```

> Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar, der Expertenmodus ist für alle, die eine größere Herausforderung suchen. In diesem Modus bekommen Spieler eineinhalb Lebensleisten und müssen diese bis zum Ende des Songs erhalten. Der erflogreiche Abschluss schaltet nicht frei außer das Recht zum Angeben.

Reviewer: ```suggestion
Am Anfang gesperrt und nur durch ein A oder besser im Streammodus freischaltbar, ist der Expertenmodus ist für alle gedacht, die eine größere Herausforderung suchen. In diesem Modus bekommen Spieler eineinhalb Lebensleisten und müssen diese bis zum Ende des Songs erhalten. Der erflogreiche Abschluss schaltet nicht frei außer das Recht zum Angeben.
```

Just a suggestion. I think this flows better.

Die Hintergrundfarbe ist violett und die Lichter verdunkeln sich, abhängig von den aktuellen Lebenspunkten

## Benutzeroberfläche

---

![](img/Os-MM.png "Hauptmenü")

---

![](img/Os-SL.jpg "Songliste")

---

![](img/Os-Store.jpg "Store")

---

![](img/Os-DS.png "Wahl des Schwierigkeitsgrad")

---

![](img/Os-SI.png "Songinformationen")

---

![](img/Os-PlayExpert.jpg "Expertenmodus")

---

![](img/Os-Pause.png "Pausenmenü")

---

![](img/Os-FM.png "Menü bei einem gescheiterten Versuch")

---

![](img/Os-SC.png "Stage cleared!")

---

![](img/Os-Grade.jpg "Ergenisbildschirm")

---

![](img/Os-O1.png "Optionsmenü 1")

---

![](img/Os-O2.png "Optionsmenü 2")

---

## Songliste

***[Klicke hier für die gesamte Liste](Song_List)***

## Punktevergabe

> *Hinweis: Die Rangliste kann noch im Spiel angesehen werden, allerdings werden neue Punktestände angenommen.*

Reviewer 3: ```suggestion
*Hinweis: Die Rangliste kann noch im Spiel angesehen werden, allerdings werden keine neuen Punktestände mehr angenommen.*
```

> *Hinweis: Die Rangliste kann noch im Spiel angesehen werden, allerdings werden neue Punktestände angenommen.*

Reviewer: ```suggestion
*Hinweis: Die Rangliste kann noch im Spiel angesehen werden, allerdings werden keine neuen Punktestände angenommen.*
```

### Genauigkeit

`Genauigkeit = Trefferwert / (perfekter Trefferwert)`

| Begriff | Formel |
| :-: | :-- |

> | **Trefferwert** | (Anzahl der Misses \* 0 + Anzahl der 50er \* 1 + Anzahl der 100er * 2 + Anazhl der 300er \* 4) |

Reviewer 2: ```suggestion
| **Trefferwert** | (Anzahl der Misses \* 0 + Anzahl der 50er \* 1 + Anzahl der 100er * 2 + Anzahl der 300er \* 4) |
```

| **perfekter Trefferwert** | (Anzahl der Misses + Anzahl der 50er + Anzahl der 100er + Anzahl der 300er) \* 4 |

Mit anderen Worten, jeder Miss ist 0%, jeder 50er ist 25%, jeder 100er ist 50% und jeder 300er ist 100% Genauigkeit wert.

### Punkte

> Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, wird mit folgender Formel berechnet:

Reviewer 3: ```suggestion
Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, werden mit folgender Formel berechnet:
```

> Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, wird mit folgender Formel berechnet:

Reviewer: ```suggestion
Die Punkte, die von jedem Hit-Circle und jedem Ende eines Sliders vergeben werden, werden mit folgender Formel berechnet:
```

Alternatively could use `Punktzahl` instead.

> `Punkte = Trefferwert + Trefferwert * (Combomultiplizierer * fester Multiplizierer) / 25`

Reviewer 3: ```suggestion
`Punkte = Trefferwert + Trefferwert * (Combo-Multiplikator * fester Multiplikator) / 25`
```

| Begriff | Bedeutung |
| :-: | :-- |
| **Trefferwert** | Die Bewertung des Hit-Circles (50, 100 oder 300). |

> | **Combomultiplizierer** | (Combo vor dem Treffer - 1) oder 0; abhängig davon, was höher ist. |

Reviewer 3: ```suggestion
| **Combo-Multiplikator** | (Combo vor dem Treffer - 1) oder 0; abhängig davon, was höher ist. |
```

> | **Fester Multiplizierer** | Der Multiplizierer der den Punktestand bei 600.000 fixiert. |

Reviewer 3: ```suggestion
| **Fester Multiplikator** | Der Multiplikator der den Punktestand bei 600.000 fixiert. |
```

> | **Fester Multiplikator** | Der Multiplikator der den Punktestand bei 600.000 fixiert. |

Reviewer 2: ```suggestion
| **Fester Multiplikator** | Der Multiplikator, der den Punktestand bei 600.000 fixiert. |
```

> Zusätzlich gibt jeder Sliderstart, -ende und -repeat tick 30 Punkte und jede volle Umdrehung eines Spinners gibt 100 Punkte.

Reviewer 3: ```suggestion
Außerdem gibt es für jeden Start-, End- und Wiederholungstick eines Sliders 30 Punkte und für jede volle Umdrehung eines Spinners 100 Punkte.
```

Es gibt einen weiteren Bonus von 10 Punkten, abhängig von den Umdrehungen pro Minute nachdem die Spinnermessbar gefüllt ist.

### Bewertung des Hit-Circles

**Hit-Circles**

> - Eine 300, 100 oder 50 wird bei einem normalen Hit-Circle verliehen, abhängig von der Treffgenauigkeit.

Reviewer 3: ```suggestion
- Ein 300er, 100er oder 50er wird bei einem normalen Hit-Circle verliehen, abhängig von der Treffgenauigkeit.
```

- Ein Miss wird vergeben, wenn ein Hit-Circle zu früh oder überhaupt nicht getroffen wurde.

>   - Wenn das Objekt viel viel zu früh getroffen wird, wackelt es stattdessen. Nichts wird ansosnten passieren, außer die Visualisierung, dass das Objekt zu früh getroffen wurde.

Reviewer 3: ```suggestion
  - Wenn das Objekt sehr, sehr früh getroffen wird, wackelt es stattdessen. Dabei passiert technisch gesehen nichts, außer dass man visuell erkennen kann, dass es zu früh angetippt wurde.
```

**Hold-Circle**

> - Die Bewertung des Hold-Circles hängt ab von dem initialen Treffer und die Haltezeit, bis der Kreis explodiert.

Reviewer 3: ```suggestion
- Die Bewertung des Hold-Circles hängt von der anfänglichen Berührung und der Haltezeit vor der Explosion ab.
```

  - 300 bei perfekter Ausführung.

>   - 100 für einen leicht inkorrekten initialen Treffer und den Kreis richtig halten.

Reviewer 3: ```suggestion
  - 100 für einen weniger präzisen anfänglichen Treffer und das richtige Halten.
```

>   - 100 für einen leicht inkorrekten initialen Treffer und den Kreis richtig halten.

Reviewer: ```suggestion
  - 100 für einen leicht inkorrekten initialen Treffer und das richtige Halten des Kreises.
```

Author: It needs to be `dem richtigen Halten des Kreises`, doesn't it?

>   - 50 für eine leichte Berührung.

Reviewer 3: ```suggestion
  - 50 dafür, dass er zumindest berührt wurde.
```

  - 30 für jede Explosion.

>   - Der Multiplizierer wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss gibt es, wenn die Kreise überhaupt nicht gehalten wurden.

Reviewer 3: ```suggestion
  - Der Multiplikator wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss gibt es, wenn die Kreise überhaupt nicht gehalten wurden.
```

>   - Der Multiplizierer wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss gibt es, wenn die Kreise überhaupt nicht gehalten wurden.

Reviewer: ```suggestion
  - Der Multiplizierer wird zurückgesetzt, wenn die Kreise nicht korrekt gehalten wurden. Ein Miss wird vergeben, wenn die Kreise überhaupt nicht gehalten wurden.
```

**Slider**

> - **Die Bewertung von Slidern hängt ab von der Genauigkeit des initialen Treffers**.

Reviewer 3: ```suggestion
- **Die Bewertung von Slidern hängt von der Genauigkeit der anfänglichen Berührung ab**.
```

- Slider bestehen aus Sliderticks, die den Start, das Ende und die repeat points des Sliders einbeziehen.

>   - Eine 300, wenn man alle Sliderticks erreicht und einen akkuraten initialen Treffer hat.

Reviewer 3: ```suggestion
  - Einen 300er, wenn man alle Sliderticks erreicht und einen genauen ersten Treffer hat.
```

>   - Eine 100, wenn mindestens die Hälfte des Sliders mit einem guten initialen Treffer getroffen wurde.

Reviewer 3: ```suggestion
  - Ein 100er, wenn mindestens die Hälfte des Sliders mit einem guten ersten Treffer getroffen wurde.
```

Author: Changed this to `Einen 100er`.

>   - Eine 50, wenn mindestens ein Slidertick oder der initiale Treffer getroffen wurde.

Reviewer 3: ```suggestion
  - Ein 50er, wenn mindestens ein Slidertick oder der erste Treffer getroffen wurde.
```

Author: Same as above.

> - **Einen Slider viel zu früh zu tippen** wird keinen Miss erzeugen, aber **wird den Punktemultiplizierer auf 0 zurücksetzen**.

Reviewer 3: ```suggestion
- **Einen Slider viel zu früh zu tippen** wird keinen Miss erzeugen, aber **wird den Punktemultiplikator auf 0 zurücksetzen**.
```

- Ein Miss, wenn der Slider überhaupt nicht gehalten wurde.

**Spinner**

- Für einen Spinner werden 300er, 100er oder 50er vergeben, abhängig von der Anzahl der gemachten Spins verglichen mit der Länge des Spinners.

> - Ein Miss, wenn die Spinnermessbar, die auf beiden Seiten des Spinners zu sehen ist, nicht gefüllt wurde.

Reviewer: ```suggestion
- Ein Miss, wenn die Spinnermessleiste, die auf beiden Seiten des Spinners zu sehen ist, nicht gefüllt wurde.
```

Might be  a personal thing but the first thing I think of when reading this is measurable spinners.

> ### Multiplizierer

Reviewer 3: ```suggestion
### Multiplikator
```

> Das Folgende erhöht den Punktemultiplizierer um einen Punkt:

Reviewer 3: ```suggestion
Das Folgende erhöht den Punktemultiplikator um einen Punkt:
```

> Das Folgende erhöht den Punktemultiplizierer um einen Punkt:

Reviewer: ```suggestion
Das Folgende erhöht den Punktemultiplizierer um jeweils einen Punkt:
```

- Treffer auf einem Hit-Circle.
- Der Start, das Ende und jeder Hold-Tick eines Hold-Circles.
- Der Start, das Ende und jeder Slidertick eines Sliders.
- Einen Spinner vervollständigen.

> Das Folgende setzt den Punktemultiplizierer auf null zurück:

Reviewer 3: ```suggestion
Das Folgende setzt den Punktemultiplikator auf null zurück:
```

- Einen Hit-Circle verfehlen.

> - Den Hold-Circle loszulassen, bevor dieser noch nicht explodiert ist.

Reviewer: ```suggestion
- Den Hold-Circle loszulassen, bevor dieser explodiert ist.
```

Alternatively `wenn dieser noch nicht explodiert ist`, but you can't have both.

> - Unfähig, die verbundenen Beats gleichmäßig zu treffen. (Einen treffen und den anderen verfehlen)

Reviewer 3: ```suggestion
- Die verbundenen Beats nicht gleichzeitig treffen. (Einen treffen und den anderen verfehlen)
```

> - Unfähig, die verbundenen Beats gleichmäßig zu treffen. (Einen treffen und den anderen verfehlen)

Reviewer: ```suggestion
- Unfähig, die verbundenen Beats gleichzeitig zu treffen. (Einen treffen und den anderen verfehlen)
```

- Einen Slider zu früh klicken.
- Den Sliderstart oder einen Slidertick verfehlen.

> - Unfähig, die Spinnermessbar vollständig zu füllen.

Reviewer 3: ```suggestion
- Die Spinnermessbar nicht vollständig füllen.
```

Author: FamousPig suggested `Spinnermessleiste`, so I will use this.

> - Unfähig, die Spinnermessbar vollständig zu füllen.

Reviewer: ```suggestion
- Unfähig, die Spinnermessleiste vollständig zu füllen.
```

> Das Folgende wird den Punktemultiplizierer weder erhöhen noch zurücksetzen:

Reviewer 3: ```suggestion
Das Folgende wird den Punktemultiplikator weder erhöhen noch zurücksetzen:
```

- Ein Sliderende verfehlen. (Es wird ein Good! oder ein 100/50 Treffer vergeben)
- Punktebonus eines Spinners

### Benotung

![](img/Os-Grade.jpg "Ergebnisbildschirm")

**Normale Noten**

- SS = 100% Genauigkeit
- S = Über 90% Genauigkeit
- A = Über 80% Genauigkeit
- B = Über 70% Genauigkeit
- C = Über 60% Genauigkeit
- D = Alles andere.

### Lebensanzeige

> Nicht wie in osu!, ist die Lebensanzeige anders in osu!stream, da sie auf dem Spielmodus basiert.

Reviewer 2: Honestly, the original sentence already is formulated in a redundant way imo. I would propose either one of the following:

```suggestion
Im Gegensatz zu osu! basiert die Lebensanzeige in osu!stream auf dem Spielmodus.
```

```suggestion
Die Lebensanzeige in osu!stream ist anders, da sie im Gegensatz zu osu! auf dem Spielmodus basiert.
```
(Closer to the original)

Im **leichten Modus ist die Lebensleiste nicht vorhanden**.

![](img/Os-SUN.jpg "Beispiel eines Stream Up! von Normal aus")

> Im **Streammodus** erhält der Spieler drei Lebensleisten und startet mit der zweiten zur Hälfte gefüllten Lebensleiste. Spieler bewegen sich zwischen diesen beiden Lebensleisten via den **Stream Up!** und **Stream Down!** Systemen, die aktiviert werden, wenn die Lebensleiste komplett gefüllt oder geleert wird. Nach einer Ankündigung wechselt das Spiel zum neuen "Stream" durch das Austauschen des Hintergrunds und einem nahtlosem Wechsel zu einem höheren oder tieferen Schwierigkeitsgrad der Beatmap sowie das Tauschen der Lebensleisten. Wenn ein Stream Down! erscheint bekommt der Spieler temporäre Unbesiegbarkeit, während die Lebensleiste zu einem tieferen Schwierigkeitsgrad wechselt.

Reviewer 3: ```suggestion
Im **Streammodus** erhält der Spieler drei Lebensleisten und startet mit der zweiten zur Hälfte gefüllten Lebensleiste. Spieler bewegen sich zwischen diesen beiden Lebensleisten via den **Stream Up!** und **Stream Down!** Systemen, die aktiviert werden, wenn die Lebensleiste komplett gefüllt oder geleert wird. Nach einer Ankündigung wechselt das Spiel zum neuen "Stream" durch das Austauschen des Hintergrunds und einem nahtlosem Wechsel zu einem höheren oder niedrigeren Schwierigkeitsgrad der Beatmap sowie das Tauschen der Lebensleisten. Wenn ein Stream Down! erscheint bekommt der Spieler temporäre Unbesiegbarkeit, während die Lebensleiste zu einem niedrigeren Schwierigkeitsgrad wechselt.
```

> Im **Streammodus** erhält der Spieler drei Lebensleisten und startet mit der zweiten zur Hälfte gefüllten Lebensleiste. Spieler bewegen sich zwischen diesen beiden Lebensleisten via den **Stream Up!** und **Stream Down!** Systemen, die aktiviert werden, wenn die Lebensleiste komplett gefüllt oder geleert wird. Nach einer Ankündigung wechselt das Spiel zum neuen "Stream" durch das Austauschen des Hintergrunds und einem nahtlosem Wechsel zu einem höheren oder niedrigeren Schwierigkeitsgrad der Beatmap sowie das Tauschen der Lebensleisten. Wenn ein Stream Down! erscheint bekommt der Spieler temporäre Unbesiegbarkeit, während die Lebensleiste zu einem niedrigeren Schwierigkeitsgrad wechselt.

Reviewer 2: ```suggestion
Im **Streammodus** erhält der Spieler drei Lebensleisten und startet mit der zweiten zur Hälfte gefüllten Lebensleiste. Spieler bewegen sich zwischen diesen beiden Lebensleisten via den **Stream Up!** und **Stream Down!** Systemen, die aktiviert werden, wenn die Lebensleiste komplett gefüllt oder geleert wird. Nach einer Ankündigung wechselt das Spiel zum neuen "Stream" durch das Austauschen des Hintergrunds und einem nahtlosem Wechsel zu einem höheren oder niedrigeren Schwierigkeitsgrad der Beatmap sowie das Tauschen der Lebensleisten. Wenn ein Stream Down! erscheint, bekommt der Spieler temporäre Unbesiegbarkeit, während die Lebensleiste zu einem niedrigeren Schwierigkeitsgrad wechselt.
```

Im **Expertenmodus wird stattdessen eine volle Lebensleiste vergeben**.

Das Folgende füllt die Lebensleiste:

- Einen Spinner und Hold-Bubbles vervollständigen.
- Ein osu!, gold good!, green good!, 300er, oder 100er treffen,
  - Bei Hit-Circles füllen bessere Bewertungen die Lebensleiste mehr. Der letzte Hit-Circle in einer farbigen Combo vergibt mehr Lebenspunkte.
  - Bei Slidern wird die Lebensleiste abhängig von den vervollständigten Teilen gefüllt, nachdem der Slider beendet wurde.

Das Folgende reduziert die Lebensleiste:

> - Eine 50 erhalten.

Reviewer 3: ```suggestion
- Einen 50er erhalten.
```

- Ein Hit-Object oder Slider verfehlen.
- Nicht in einem Follow-Circle eines Sliders bleiben.
- Nicht die Hold-Bubble halten, bis sie explodiert.
- Einen Spinner nicht drehen.
- Einen Spinner nicht mit einer "Clear" Nachricht vervollständigen.

Im Gegensatz zu anderen Versionen von osu! reduziert sich die Lebensleiste hier nicht von selbst.

## Fragen

### Warum ist (füge hier irgendeinen urheberrechtlich geschützten Song ein) nicht enthalten?

> Wegen der Schwierigkeit die Lizenz für den oben genannten Song zu bekommen.

Reviewer 3: ```suggestion
Wegen der Schwierigkeit eine Lizenz für den oben genannten Song zu bekommen.
```

> Wegen der Schwierigkeit eine Lizenz für den oben genannten Song zu bekommen.

Reviewer 2: ```suggestion
Wegen der Schwierigkeit, eine Lizenz für den oben genannten Song zu bekommen.
```
I don't know when optional commas are correct. Shouldn't be wrong to put it here.

### Wie komme ich an mehr Beatmaps?

Gehe zu der "Store" Sektion im Spiel oder drücke den "Download more songs..." Knopf in der "Play" Sektion.

### Was ist die Titelmelodie von osu!stream?

[nekodex - osu!stream theme (1.48)](https://soundcloud.com/nekodex/osu-stream-theme).
Meistens im Hauptmenü und im Tutorial zu hören.

> ### Danksagungen?

Reviewer 3: ```suggestion
### Credits?
```
i think just using credits fits better here as its a bit more than just saying thanks

> ![](img/Os-Credit.jpg "Danksagungen")

Reviewer 3: ```suggestion
![](img/Os-Credit.jpg "Credits")
```
